build.sbt

```diff
@@ -851,13 +835,14 @@ lazy val `graal-tools` = (project in file("graal-tools"))
   .enablePlugins(GraalVMPlugin)
   .settings(
     libraryDependencies ++= List(
-        "org.graalvm.nativeimage" % "svm" % GraalVersion,
+        "org.graalvm.sdk" % "graal-sdk" % GraalVersion % "provided",
```

Comment:
Huh.  What happened here to bring this back?

Reply:
Ah, good question, I had to add that when trying the 20.1.0-dev build since it wasn't officially published anywhere. Can remove this now.